src/Resources/views/CRUD/list_outer_rows_mosaic.html.twig

```diff
@@ -30,9 +30,11 @@ This template can be customized to match your needs. You should only extends the
 
                             <div class="mosaic-inner-box-default">
                                 {% block sonata_mosaic_background %}
-                                    <img src="{{ meta.isImageAvailable is defined and not meta.isImageAvailable ?
-                                        sonata_admin.getMosaicBackground() :
-                                        meta.image }}" alt="{{ meta.title }}" />
+                                    {% set metaImage = meta.isImageAvailable is defined and not meta.isImageAvailable ? sonata_admin.mosaicBackground : meta.image %}
+                                    {% if not (metaImage starts with 'data:') %}
```

Comment:
Should we trigger a deprecation for the case `metaImage starts with 'data:'` and removing it in the next major ?

Reply:
I didn't see a problem of leaving support in, in case anyone wants to genuinely use data URI's.

Best would be to have `asset()` accept data URI's and process like absolute URLs by returning them as-is, but waiting before that change happens takes too long.

If you and more people think data URI's should be deprecated that is fine by me.